src/NHibernate/Util/LinkedHashMap.cs

```diff
@@ -19,6 +19,8 @@ namespace NHibernate.Util
 	/// </remarks>
 	[DebuggerTypeProxy(typeof(CollectionProxy<>))]
 	[Serializable]
+	// Since 5.5
+	[Obsolete("This class has no more usages and will be removed in a future version. Replaced with internal LinkHashMap class")]
```

Comment:
It will be 5.6. No need to mention the replacement since it will not be available for external use.

```suggestion
	// Since 5.6
	[Obsolete("This class has no more usages and will be removed in a future version.")]
```

Reply:
applied